sonar-project.properties

```diff
@@ -0,0 +1,28 @@
+sonar.projectKey=openshift_odo
+sonar.projectName=odo
+
+# Path to sources
+sonar.sources=cmd,pkg
+# Source File Exclusions: Patterns used to exclude some source files from analysis.
+# sonar.exclusions=
```

Comment:
Don't you want to exclude the tests files from the pkg and cmd directories?

Reply:
So this the part where I was confused. Earlier I excluded test files from the source directory, but then we had unit tests that I did not want to miss out on, so I removed them from the exclusion. But there, fixed it now.

See - https://sonarcloud.io/documentation/project-administration/narrowing-the-focus/ to understand the changes that I made.